src/ledger/LedgerManagerImpl.cpp

```diff
@@ -1725,8 +1726,9 @@ LedgerManagerImpl::transferLedgerEntriesToBucketList(
     ltx.getAllEntries(initEntries, liveEntries, deadEntries);
     if (blEnabled)
     {
-        mApp.getBucketManager().addBatch(mApp, lh, initEntries, liveEntries,
-                                         deadEntries);
+        BucketTestUtils::addBatchAndUpdateSnapshot(
```

Comment:
This looks really suspicious, how come we're using a test utility in (seemingly) the prod code path?

Reply:
You're right, I messed up the rebase, this is supposed to be on the test-only variant,  `LedgerManagerForTesting::transferLedgerEntriesToBucketList`